components/locid/src/extensions/transform/value.rs

```diff
@@ -26,8 +26,8 @@ use tinystr::TinyAsciiStr;
 /// let value2: Value =
 ///     "hybrid-foobar".parse().expect("Failed to parse a Value.");
 ///
-/// assert_eq!(&value1.to_string(), "hybrid");
-/// assert_eq!(&value2.to_string(), "hybrid-foobar");
+/// assert_eq!(value1, "hybrid".parse().unwrap());
+/// assert_eq!(value2, "hybrid-foobar".parse().unwrap());
```

Comment:
Observation: this is a vacuous assertion, because the two values were created with the same `parse` function